src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/documentmapper/PartADocumentMapper.kt

```diff
@@ -119,17 +119,20 @@ class PartADocumentMapper : RecommendationDataToDocumentMapper() {
       riskToKnownAdult = recommendation.currentRoshForPartA?.riskToKnownAdult?.partADisplayValue,
       riskToStaff = recommendation.currentRoshForPartA?.riskToStaff?.partADisplayValue,
       riskToPrisoners = recommendation.currentRoshForPartA?.riskToPrisoners?.partADisplayValue,
-      countersignAcoEmail = recommendation.countersignAcoEmail,
-      counterSignSpoEmail = recommendation.countersignSpoEmail,
-      countersignSpoName = recommendation.countersignSpoName,
-      countersignSpoTelephone = recommendation.countersignSpoTelephone,
+
+      countersignAcoEmail = metadata.acoCounterSignEmail,
```

Comment:
I think it would be nice if these PO/SPO/ACO values were sorted so that they were all together by "person" and then within that group, in the same order. (i.e. we would have the practitioner, the SPO then the ACO and it would always be name, email, telephone etc.). That makes it easier to see what is present and what is not.

Reply:
Good idea!